src/components/annual-report/report-data/ReportData.module.css

```diff
@@ -0,0 +1,13 @@
+.reportContainer {
+    margin: 16px;
+}
+
+.sectionHeading {
+    display: block;
+    background: #90caf9; /*blue300*/
```

Comment:
seems like the old one is using `#5b92e5` as background color.  the one uses is a bit brighter .